jekyll/_cci2/status-badges.md

```diff
@@ -5,28 +5,43 @@ title: Embedding Build Status Badges
 
 [Basics]({{ site.baseurl }}/2.0/basics/) > Embedding Build Status Badges
 
-This document describes how to embed a badge that shows the build status (passed or failed) for your project in your README or in another document. You can generate code for the following formats: Image URL, Markdown, Textile, Rdoc, AsciiDoc, reStructuredText, and pod. 
+This document describes how to create a badge that displays your project's build status.
```

Comment:
Again, I don't understand the rewrite of this intro to include less information. Leave as-is please.

Reply:
The information is included in the _Overview_ section below. In my mind, the introduction should be as brief as possible, letting the _Overview_ do the bulk of the explaining.

Maybe I'm confused about the purposes of the introduction and the overview. Could you clarify what they should be for? And should every document have both?

Reply:
I see, I think the original intro sentence is better than the one in the Overview with the use of the word 'virtually' which could be confusing and because it doesn't say what status values are possible (it just says 'build status' which might not be clear to some users). Yes, each article needs both and they should ideally have different information in them, the first sentence tells you why you would read the document (this document describes xyz), the second is a conceptual overview of the topic.